csrc/mma_type.h

```diff
@@ -210,7 +210,7 @@ enum class MmaLayout { NT = 0, TT, TN, NN };
 enum class UnitDim { K, M_or_N };
 
 //! Utility to annotate which input of mma this option struct describes
-enum class MmaOperand { Accumulator = 0, A, B };
+enum class MmaOperand { A, B };
```

Comment:
Off-topic: could we instead just use `MatmulTensorRole`?

Reply:
I think we should. Let's create a separate PR to clean this up after this PR is being merged.